core/common/src/main/java/alluxio/security/authentication/ClientContextServerInjector.java

```diff
@@ -11,39 +11,59 @@
 
 package alluxio.security.authentication;
 
+import alluxio.grpc.ClientVersionClientInjector;
+
 import io.grpc.ForwardingServerCallListener;
 import io.grpc.Grpc;
 import io.grpc.Metadata;
 import io.grpc.ServerCall;
 import io.grpc.ServerCallHandler;
 import io.grpc.ServerInterceptor;
 
+import javax.annotation.Nullable;
+
 /**
  * Server side interceptor that is used to put remote client's IP Address to thread local storage.
  */
-public class ClientIpAddressInjector implements ServerInterceptor {
+public class ClientContextServerInjector implements ServerInterceptor {
 
   /**
    * A {@link ThreadLocal} variable to maintain the client's IP address along with a specific
    * thread.
    */
-  private static ThreadLocal<String> sIpAddressThreadLocal = new ThreadLocal<>();
+  private static final ThreadLocal<String> IP_ADDRESS_THREAD_LOCAL = new ThreadLocal<>();
+  /**
+   * A {@link ThreadLocal} variable to maintain the client's version along with a specific
+   * thread.
+   */
+  private static final ThreadLocal<String> CLIENT_VERSION_THREAD_LOCAL =
```

Comment:
i think we name private state variables like `sIpAddressThreadLocal ` and public static vars like `IP_ADDRESS_THREAD_LOCAL`

Reply:
I think it's not about whether it's private, it's about whether it's final.